x-pack/plugins/index_lifecycle_management/public/application/sections/policy_table/policy_table.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, ReactElement, ReactNode, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiFieldSearch,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiPageBody,
+  EuiPageContent,
+} from '@elastic/eui';
+import { ApplicationStart } from 'kibana/public';
+import { RouteComponentProps } from 'react-router-dom';
+import { reactRouterNavigate } from '../../../../../../../src/plugins/kibana_react/public';
+import { PolicyFromES } from '../../services/policies/types';
+import { filterItems } from '../../services';
+import { TableContent } from './components/table_content';
+
+interface Props {
+  policies: PolicyFromES[];
+  history: RouteComponentProps['history'];
+  navigateToApp: ApplicationStart['navigateToApp'];
+  updatePolicies: () => void;
+}
+
+export const PolicyTable: React.FunctionComponent<Props> = ({
+  policies,
+  history,
+  navigateToApp,
+  updatePolicies,
+}) => {
+  const [confirmModal, setConfirmModal] = useState<ReactNode | null>();
+  const [filter, setFilter] = useState<string>();
+  const [selectedPolicies] = useState<PolicyFromES[]>([]);
```

Comment:
Do we never set this piece of state?

Reply:
Good catch, JL! This was left over from old code and I was thinking that we have a checkbox at the beginning of a row to select policies (to delete several at once, for example). But it was actually not used, so I'm deleting `selectedPolicies` for now. We might add those checkboxes later.